### PR TITLE
rpi3 ll40ls driver

### PR DESCRIPTION
**Pull request by belaidH**

### CODE REVIEW

File: src/drivers/ll40ls/LidarLiteI2C.cpp [hunk 1]
```diff
@@ -225,10 +225,10 @@ int LidarLiteI2C::ioctl(struct file *filp, int cmd, unsigned long arg)
 		return _reports->size();
 
 	default: {
-			int result = LidarLite::ioctl(filp, cmd, arg);
+			int result = LidarLiteI2C::ioctl(filp, cmd, arg);
 
 			if (result == -EINVAL) {
```
Comment (dagar) [inline src/drivers/ll40ls/LidarLiteI2C.cpp]: You can get rid of this. LiderLiteI2C::ioctl() will call I2C::ioctl().

File: src/drivers/ll40ls/LidarLiteI2C.cpp [hunk 2]
```diff
@@ -225,10 +225,10 @@ int LidarLiteI2C::ioctl(struct file *filp, int cmd, unsigned long arg)
...
-				result = I2C::ioctl(filp, cmd, arg);
+				result = LidarLiteI2C::ioctl(filp, cmd, arg);
```
Comment (dagar) [inline src/drivers/ll40ls/LidarLiteI2C.cpp]: You can get rid of this. LiderLiteI2C::ioctl() will call I2C::ioctl().

**Comment by dagar** [comment]
Looking good. Did you see the raspberrypi build failure here? http://ci.px4.io:8080/blue/organizations/jenkins/Firmware/detail/PR-8579/1/pipeline

Replace `struct file` with `file_t`.

**Comment by belaidH** [comment]
@dagar thanks, I think i need to include file because I have changed all the "struct file" into "file_t" but I get error that says "file_t has not been declared".

capture du 2018-01-04 23-38-31

**Comment by dagar** [comment]
Continued in https://github.com/PX4/Firmware/pull/8633

**Pull request closed without merging**